common/i18n/resources/en.js

```diff
@@ -58,14 +59,17 @@ module.exports = {
       deleteWarining: 'Deleting the **{{surveyName}}** survey will delete all of its data.',
       confirmName: 'Enter this survey’s name to confirm:',
     },
-    surveyListView: {
+    surveyListTable: {
       addNewSurvey: 'Add a new Survey',
-      name: 'Name',
-      label: 'Label',
       dateCreated: 'Date created',
       dateLastModified: 'Date last modified',
       status: 'Status',
     },
+    surveyCreate: {
+      createSurvey: 'Create Survey',
+      importFromCollect: 'Import from Collect',
+      language: 'Language',
```

Comment:
language is quite a generic word, (and after moving placeholder - see LanguageDropdown comment). i would move this in common